michigan_week1/MinimumCostPath.java

```diff
@@ -0,0 +1,36 @@
+package src;
+
+public class MinimumCostPath {
+
+    int minimumCost(int[][] cityDistancesGraph, int sourceCity, int destinationCity)
```

Comment:
method that returns something should have get appended to it. for void you can have the name that you have written